scripts/ripe-atlas

```diff
@@ -1,94 +1,10 @@
 #!/usr/bin/env python
-
-import importlib
-import os
-import pkgutil
-import re
 import sys
 
-from ripe.atlas.tools import commands
+from ripe.atlas.tools.script import RipeAtlas
```

Comment:
This is in anticipation of building bundles using pyinstaller, which has the concept of import hooks for discovering hidden imports. It may be possible to get it working with the script as it is, but I was taking my cue from stuff like https://github.com/django/django/blob/master/django/bin/django-admin.py